package Exceptions.TryCatch;

public class Main {
    public static void main(String[] args) throws StudentException {

        Student s1 = Student.find("21");

    }
}
